syft/frameworks/torch/he/fv/evaluator.py

```diff
@@ -58,6 +62,18 @@ def negate(self, ct):
 
         return CipherText(result)
 
+    def mul(self, op1, op2):
+        """Multiply two operands using FV scheme.
+
+        Args:
+            op1 (Ciphertext/Plaintext): First argument.
+            op2 (Ciphertext/Plaintext): Second argument.
+
+        Returns:
+            A Ciphertext object with a value equivalent to the result of the product of two
+                operands.
+        """
+
```

Comment:
I have to wait for #3775 to merge before completing this method because the way of checking the operand type changed in that PR. otherwise, this PR is ready for review.